examples/config.py

```diff
@@ -0,0 +1,159 @@
+#!/usr/bin/env python3
+
+#                        openNetVM
+#                https://sdnfv.github.io
+#
+# OpenNetVM is distributed under the following BSD LICENSE:
+#
+# Copyright(c)
+#       2015-2018 George Washington University
+#       2015-2018 University of California Riverside
+# All rights reserved.
+#
+# Redistribution and use in source and binary forms, with or without
+# modification, are permitted provided that the following conditions
+# are met:
+#
+# * Redistributions of source code must retain the above copyright
+#   notice, this list of conditions and the following disclaimer.
+# * Redistributions in binary form must reproduce the above copyright
+#   notice, this list of conditions and the following disclaimer in
+#   the documentation and/or other materials provided with the
+#   distribution.
+# * The name of the author may not be used to endorse or promote
+#   products derived from this software without specific prior
+#   written permission.
+#
+# THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+# "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT
+# LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR
+# A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT
+# OWNER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL,
+# SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT
+# LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE,
+# DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY
+# THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+# (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+# OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+
+"""This script allows a user to launch a chain of NFs from a JSON
+config file passed in as an argument when running the script"""
+
+import sys
+import json
+import os
+import shlex
+import time
+from signal import signal, SIGINT
+from subprocess import Popen
+from datetime import datetime
+
+def handler(signal_received, frame):
+    """Handles cleanup on user shutdown"""
+    # pylint: disable=unused-argument
```

Comment:
Sorry for the nit but I knew this was a thing in Python https://stackoverflow.com/a/54213922/10158041

```suggestion
def handler(signal_received, _):
    """Handles cleanup on user shutdown"""
```

Reply:
When I make this fix, I'm still getting a linter error for unused argument "signal_received". If I change both arguments to "_", I get linter errors for duplicate argument names. Not sure if there's a fix for this?